outport/process/alteredaccounts/alteredAccountsProvider.go

```diff
@@ -230,20 +235,25 @@ func (aap *alteredAccountsProvider) addTokensDataForMarkedAccount(
 	return nil
 }
 
-func (aap *alteredAccountsProvider) convertMetaData(metaData *esdt.MetaData) *outportcore.TokenMetaData {
+func (aap *alteredAccountsProvider) convertMetaData(metaData *esdt.MetaData) (*outportcore.TokenMetaData, error) {
 	if metaData == nil {
-		return nil
+		return nil, nil
```

Comment:
this is extremely dangerous. Let's refactor. 
Reason: you test for nil error and then you want to use the object (that might also be nil)

Reply:
refactored, my bad